airbyte-config/specs/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
 
     implementation project(':airbyte-commons')
     implementation project(':airbyte-commons-cli')
+    implementation project(':airbyte-commons-docker')
```

Comment:
This was added so that we can use the DockerUtils to get the tagged docker image name